### PR TITLE
[python] update README.md

### PR DESCRIPTION
**Pull request by yawpitch**
Updates the readme to reflect the state of Python's Analyzer, Representer, and Test Runner.

### CODE REVIEW

File: languages/python/README.md [hunk 1]
```diff
@@ -51,5 +51,5 @@ These extra steps will make your track better, but are optional.
 
 ### Analyzer
 
-- [ ] Build Analyzer
+- [x] Build [Analyzer](https://github.com/exercism/python-analyzer)
 - [ ] Deploy Analyzer
```
Comment (ErikSchierboom) [inline languages/python/README.md]: Isn't the Analyzer deployed too? As in, it has already commented on v2 exercises?

File: languages/python/README.md [hunk 2]
```diff
@@ -37,12 +37,12 @@ Before launch, we need all of the following parts to be completed:
...
 
-- [ ] Build Representer
+- [x] Build [Representer](https://github.com/exercism/python-representer)
 - [ ] Deploy Representer
 
 ### Test Runner
 
-- [ ] Build Test Runner
+- [x] Build [Test Runner](https://github.com/exercism/python-test-runner)
 - [ ] Deploy Test Runner
```
Comment (ErikSchierboom) [inline languages/python/README.md]: According to [this list](https://github.com/exercism/automated-tests/issues/32). the Python test runner is also working, so I think you can consider it deployed.

File: languages/python/README.md (hunk 1 above)
Comment (yawpitch) [reply languages/python/README.md]: Yes, I wasn't clear on whether or not "deployed" in this case meant to some v3 infrastructure, or currently in use for V2 or the research experiment.

File: languages/python/README.md (hunk 2 above)
Comment (yawpitch) [reply languages/python/README.md]: I'll tick both.

File: languages/python/README.md (hunk 1 above)
Comment (ErikSchierboom) [reply languages/python/README.md]: I understand. You can definitely consider this "deployed."

File: languages/python/README.md (hunk 2 above)
Comment (ErikSchierboom) [reply languages/python/README.md]: Great!

File: languages/python/README.md (hunk 1 above)
Comment (yawpitch) [reply languages/python/README.md]: Cool. Ticked.

**Pull request merged**